src/components/videos/videoCard.js

```diff
@@ -0,0 +1,27 @@
+import { Link } from "react-router-dom";
+import { AiFillLike } from "react-icons/ai";
+import { MdWatchLater, MdPlaylistAdd } from "react-icons/md";
+
+export const VideoCard = ({ _id, title, creator, views }) => {
+    return (
+        <Link to={`/singleVideo/${_id}`} className="video-link">
+            <div class="ui-component card card-with-badge">
+                <div class="card-image">
+                    <img src={`https://img.youtube.com/vi/${_id}/hqdefault.jpg`} />
```

Comment:
you can add an `alt` attribute for image, so that if image doesn't loads the text appears

Reply:
Ok, thank you